doc/rados/operations/change-mon-elections.rst

```diff
@@ -0,0 +1,65 @@
+.. _changing_monitor_elections:
+
+==================================
+Configure Monitor Election Strategies
+==================================
+
+By default, the monitors will use the CLASSIC option it has always used. We
+recommnd you stay in this mode unless you require features in the other
+modes.
+
+If you want to switch modes BEFORE constructing the cluster, change
+the ``mon election default strategy`` option. This option is an integer value:
+
+* 1 for CLASSIC
+* 2 for DISALLOW
+* 3 for CONNECTIVITY
+
+Once your cluster is running, you can change strategies by running ::
+
+  $ ceph mon set election strategy {CLASSIC|DISALLOW|CONNECTIVITY}
+
+Choosing a mode
+===============
+The modes other than CLASSIC provide different features. We recommend
+you stay in CLASSIC mode if you don't need the extra features as it is
+the simplest mode.
+
+The DISALLOW Mode
+=================
+This mode lets you name monitors as DISALLOWED, in which case they will
+participate in the quorum and serve clients, but cannot be elected leader. You
+may wish to use this if you have some monitors which are known to be far away
+from clients 
+You can disallow a leader by running ::
+
+  $ ceph mon add disallowed leader {name}
+
+You can remove a monitor from the disallowed list, and allow it to become
+a leader again, by running ::
+
+  $ ceph mon rm disallowed leader {name}
+
+The CONNECTIVITY Mode
+=====================
+This mode evaluates connection scores provided by each monitor for its
+peers and elects the monitor with the best score. This mode is designed
+to handle netsplits, which may happen if your cluster is stretched across
```

Comment:
s/netsplits/asymmetrical netsplits/?

Reply:
I don't think that's accurate. If the live connections are A-B, A-C, B-C (all bidirectional) you will still permanently cycle as both A and B send propose messages to C and it spawns an election to try and bring the out-of-quorum monitors in.

I've actually seen this happen *once* in the wild, although I think in that case it was the network card or a bug somewhere inside the server rather than in the network infrastructure itself.